registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -247,6 +248,7 @@
     "sha256:3f791c0c0ead3c788723030563f1ae4ce9eae7a88cff9f9293ef81b1e3a21dc1": ["v1.11.0-beta.0"]
     "sha256:511dd068ccd2777af4ffc6247b5d4120a2f2c9653cab6bb1487382ee5ba59513": ["v1.11.0-beta.1"]
     "sha256:8665d717c2b0ddcfc35b08f895c6d3f3e4d5ba3abd1c3ead13394015e012c842": ["v1.11.0-beta.2"]
+    "sha256:8a84e4e228183193c9ec73cefb5a99016d7f0797240d7c72a71ea4d6797f2ac9": ["v1.11.0-rc.0"]
```

Comment:
SHA `8a84e4e22818` matches clusterctl v1.11.0-rc.0 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:8a84e4e228183193c9ec73cefb5a99016d7f0797240d7c72a71ea4d6797f2ac9/details?inv=1&invt=Ab4ryw).